apps/studio/src/index.tsx

```diff
@@ -14,6 +15,9 @@ if (window.location.hostname.includes("newm-studio")) {
 initializeSentry();
 loadReferralHeroScript();
 
+const browserLocale = navigator.language.toLowerCase();
+moment.locale(browserLocale);
```

Comment:
Properly formats moment based calls to use locale formatting. the library moment.js does not automatically grab the browser's locale. The default was 'en' which ends up always displaying United States formatting rules, for example when `date.format("L")` is used. 

Moving forward instead of importing as `import moment from "moment";` we should instead use `import moment from "moment/min/moment-with-locales";`. Testing should also handle multiple locale options to confirm date formats appear as they should.